### PR TITLE
feat: feeds and general improvements

### PR DESCRIPTION
**Pull request by AuHau**

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -118,7 +118,8 @@ h1{
   background: #2CAA5F;
 }
 
-.alert--tip{
+/* a.k.a as Tip admonition*/
```
Comment (vojtechsimetka) [inline src/css/custom.css:121]: ?

File: docs/user-documentation/upload-download.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ console.log(retrievedFile.data.text()) // prints 'Bee is awesome!'
   </TabItem>
 </Tabs>
 
-In browsers, you can upload directly `File` type. The filename is taken from the file object itself, but can be overwritten through the second argument of the `uploadFile` function (see the [API docs](./api/classes/bee#uploadfile))
+In browsers, you can upload directly `File` type. The filename is taken from the file object itself, but can be overwritten through the second argument of the `uploadFile` function (see the [API docs](../api/classes/bee#uploadfile))
```
Comment (vojtechsimetka) [inline docs/user-documentation/upload-download.md:106]:
```suggestion
In browsers, you can upload directly `File` type. The filename is taken from the file object itself, but can be overwritten through the second argument of the `uploadFile` function (see the [API docs](../api/classes/bee.md#uploadfile))
```

**Review by vojtechsimetka — APPROVED** [review]

File: src/css/custom.css (hunk 1 above)
Comment (AuHau) [reply src/css/custom.css:121]: Well in here it is called "success" but in Docusaurus it is called "Tip" so just a note :-)

File: docs/user-documentation/getting-started.md [hunk 3]
```diff
@@ -76,3 +76,16 @@ That’s it! now you can use the `bee` object.
...
+
+:::tip Using `<script>` import
+
+If you include `bee-js` using the `unpkg.com` script link then all the exported components will be available to you
+under global namespace `BeeJs`:
+
+```html
+<script src="https://unpkg.com/@ethersphere/bee-js/dist/index.browser.min.js"></script>
+<script>
+  const bee = new BeeJs.Bee('...')
```
Comment (agazso) [inline docs/user-documentation/getting-started.md:88]: Thanks, this was very much needed!

**Review by agazso — APPROVED** [review]
Looks good!

**Pull request merged**